ambuda/seed/lookup/user_permissions.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy.orm import Session
```

Comment:
Generally, database seeding scripts write a small amount of fixed data to the database. This file is more analogous to a migration and should live in the migrations script, IMO.